myTrip/photo/apps.py

```diff
@@ -0,0 +1,7 @@
+"""."""
```

Comment:
delete this line
pylint ignore=CVS, migrations, apps.py, urls.py, wsgi.py. local_setings.py, manage.py